lib/Conversation/CommentForm/index.js

```diff
@@ -58,8 +60,10 @@ class CommentForm extends Component {
     if (e) {
       e.preventDefault();
     }
-    this.props.onSubmit(this.state.inputValue);
-    const inputValue = !this.props.isSubmitting ? '' : this.state.inputValue;
+    this.props.onSubmit(this.removeMentionMarkup());
```

Comment:
It might be worth doing `this.removeMentionMarkup()` and storing the value rather than doing it twice when `!this.props.isSubmitting` is true.